src/databricks/sql/auth/auth.py

```diff
@@ -70,6 +71,7 @@ def get_auth_provider(cfg: ClientContext):
 
 PYSQL_OAUTH_SCOPES = ["sql", "offline_access"]
 PYSQL_OAUTH_CLIENT_ID = "databricks-sql-python"
+PYSQL_OAUTH_AZURE_EXPERIMENTAL_CLIENT_ID = "a743d78c-536a-4ffc-b110-edfb231e90dc"
```

Comment:
is it Azure Enterprise Application? it's not visible in the AAD.

Reply:
Updated the client_id with the published Azure app ID